modules/index/src/test/java/org/apache/ignite/internal/index/ChangeIndexStatusTaskControllerTest.java

```diff
@@ -188,11 +189,13 @@ private void setPrimaryReplicaAnotherNode() {
     }
 
     private void setPrimaryReplica(ClusterNode clusterNode) {
-        TablePartitionId groupId = new TablePartitionId(tableId(), 0);
+        ZonePartitionId zonePartId = new ZonePartitionId(0, 0, tableId());
```

Comment:
Here and there. Using 0 is generally not consistent. `tableId` internally wil retrieve id from CatalogTableDescriptor 
```
    public static CatalogTableDescriptor getTableStrict(CatalogService catalogService, String tableName, long timestamp) {
        CatalogTableDescriptor table = catalogService.table(tableName, timestamp);

        assertNotNull(table, "tableName=" + tableName + ", timestamp=" + timestamp);

        return table;
    }
```
Meaning that we should retrieve zoneId from the same descriptor.